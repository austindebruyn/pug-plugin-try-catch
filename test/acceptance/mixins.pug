mixin workingMixin(x)
  div.from-mixin= x

mixin brokenMixin(x)
  div.from-mixin= x.some.thing

try
  +workingMixin("yeah")
catch
  +workingMixin("nope")

try
  +workingMixin("yeah...")
  +brokenMixin("nope")
catch
  +workingMixin("it works")
