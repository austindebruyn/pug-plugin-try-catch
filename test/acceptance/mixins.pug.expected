<div class="from-mixin">yeah</div><div class="from-mixin">it works</div>
